### PR TITLE
Added tflint_init input

### LINKED ISSUE #45: It would be helpful to run tflint init
The current method of being able to specifically call installations for plugins is great, but tflint already supports downloading plugins and has a robust system for configuring plugins and tflint itself. Use of `.tflint.hcl` means that I can have similar configurations when multiple developers are running tflint locally, and the ability to inherit that configuration (including version pinning and other configuration for plugins) by letting us call `tflint init` as a(n optional) preparatory step would be great.

### PR DESCRIPTION
**Pull request by solacelost**
- [x] Added new tflint_init input to action and readme
- [x] Added to script to run tflint_init right after running ruleset installation (so if a pinned version in `.tflint.hcl` would override the latest version, that would be preferred)
- [x] Added tests

fixes #45

### CODE REVIEW

**Comment by solacelost** [comment]
A manual test run to make sure it's doing what I expect now:

```shell
$ CI_REPO_OWNER=solacelost CI_REPO_NAME=action-tflint CI_COMMIT=d8fd91a079524e9f096dd9aa06afd753ba384558 REVIEWDOG_VERSION=v0.13.1 GITHUB_WORKSPACE=$(pwd) INPUT_GITHUB_TOKEN=<REDACTED> INPUT_LEVEL=info INPUT_REPORTER=local INPUT_FILTER_MODE=nofilter INPUT_FAIL_ON_ERROR=false INPUT_WORKING_DIRECTORY=tests/init INPUT_TFLINT_VERSION=latest INPUT_TFLINT_RULESETS='' INPUT_TFLINT_INIT=true INPUT_FLAGS=--module ./script.sh
::group::Preparing
Detected linux running on amd64, will install tools in /tmp/tmp.jdxv0iUKYK
Looking up the latest tflint version ...
::endgroup::
::group::🐶 Installing reviewdog (v0.13.1) ... https://github.com/reviewdog/reviewdog
reviewdog/reviewdog info checking GitHub for tag 'v0.13.1'
reviewdog/reviewdog info found version: 0.13.1 for v0.13.1/Linux/x86_64
reviewdog/reviewdog info installed /tmp/tmp.jdxv0iUKYK/reviewdog/reviewdog
::endgroup::
::group:: Installing tflint (v0.34.1) ... https://github.com/terraform-linters/tflint
Archive:  /tmp/tmp.jdxv0iUKYK/tflint.zip
  inflating: /tmp/tmp.jdxv0iUKYK/temp-tflint/tflint
::endgroup::
::group:: Initialize tflint from local configuration
Installing `azurerm` plugin...
Installed `azurerm` (source: github.com/terraform-linters/tflint-ruleset-azurerm, version: 0.14.0)
::endgroup::
::group:: Print tflint details ...
TFLint version 0.34.1
+ ruleset.azurerm (0.14.0)
::endgroup::
::group:: Running tflint with reviewdog 🐶 ...
example.tf:2:13: error: "Standard_DS1_v3" is an invalid value as vm_size ()
::set-output name=tflint-return-code::2
::set-output name=reviewdog-return-code::0
::endgroup::
```

**Comment by solacelost** [comment]
And a quick showing of it working successfully, where I couldn't get `rulesets` to behave as I expected given my tflint config:

https://github.com/jharmison-redhat/disconnected-openshift-testbed/runs/4668562034

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -164,6 +164,47 @@ jobs:
             exit 1
           fi
 
+  test-init:
+    if: github.event_name == 'pull_request'
```
Comment (haya14busa) [inline .github/workflows/tests.yml]:
```suggestion
```

The should work with non pull_request events.

**Review by haya14busa — APPROVED** [review]

**Comment by review-dog** [comment]
Hi, @solacelost! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintenance of reviewdog.

Thanks again!

**Comment by haya14busa** [comment]
Thank you for your contribution and sry for the late review !

**Pull request merged**